src/main/java/dev/morling/kccli/command/SetContextCommand.java

```diff
@@ -15,20 +15,32 @@
  */
 package dev.morling.kccli.command;
 
+import dev.morling.kccli.service.Context;
 import dev.morling.kccli.util.ConfigurationContext;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+import java.net.URI;
 
 @Command(name = "set-context", description = "Set the context to cluster provided in arguments")
 public class SetContextCommand implements Runnable {
+    @Parameters(index = "0", description = "Context name")
+    String contextName;
 
-    @Option(names = { "--cluster" }, description = "URL of the Kafka Connect cluster to connect to", required = true)
+    @Option(names = {"--cluster"}, description = "URL of the Kafka Connect cluster to connect to", required = true)
```

Comment:
Should we default the options? http://localhost:8083 and localhost:9092 seem reasonable for KC and broker URLs? For the offset topic, we could leave that optional. If if it's not set for a given context, the future commands interacting with offsets wouldn't work without it, stating the need to specify a value in the chosen context. WDYT?

Reply:
Done.